### DISCUSSION TITLE
Run old version of lighthouse on command line

### DISCUSSION THREAD

**Discussion by SusanaRR** [GoogleChrome/lighthouse#14803] [General] [answered]
Hi there,

I was trying to check if there is any way to run an old version of lighthouse on the command line.

If for whatever reason you need to reinstall lighthouse you will get always the new version. I would like to being able to control when the new version is installed if possible and in the meantime keep a version.

Also, when running LH on the command line can it be that the version used is a previous version as a bug??

Thanks!!!!

**Comment by connorjclark** [comment, marked as answer]
You can explicitly install it (for example, globally: `npm install -g lighthouse@9.6.0`)

Or run it like so: `npx lighthouse@9.6.0 https://www.example.com --view` (note, you need to not be inside a npm project directory, otherwise it defers to the local lighthouse version if specified in a package.json)

**Comment by SusanaRR** [comment]
Hi @connorjclark 

It did work, many thanks!!!!
